module1/homework/grayscaleImages/compression.cpp

```diff
@@ -0,0 +1,60 @@
+#include "compression.hpp"
+
+#include <iostream>
+
+void printMap(const std::array<std::array<uint8_t, width>, height>& bitmap) {
+    for (const auto& arr : bitmap) {
+        for (auto& el : arr) {
+            if (el <= ASCIIno32)
+                std::cout << " ";
+            else
+                std::cout << el;
+        }
+        std::cout << "\n";
+    }
+}
+
+std::vector<std::pair<uint8_t, uint8_t>> compressGrayscale(const std::array<std::array<uint8_t, width>, height>& grayScale) {
+    std::vector<std::pair<uint8_t, uint8_t>> compressed;
```

Comment:
enter before arguments will make your code more readable, aliases for long type are also good solution